editor/block-switcher/style.scss

```diff
@@ -34,6 +34,12 @@
 	input {
```

Comment:
Unrelated but this style is probably useless, I can't see any `input` inside the block switcher menu